dashboard/config/locales/en.yml

```diff
@@ -1226,3 +1226,22 @@ en:
       cannot_view_locked_answers: "Cannot view answers while locked"
       forbidden: "User cannot update student's lockable state"
   reset_predict_progress_button: "Delete Answer"
+  pixelation:
+    raw_format: "Raw Format"
+    raw_format_alt: "View bits without any formatting"
+    readable_format: "Readable Format"
+    readable_format_alt: "View bits separated by pixel"
+    save_image: "Save Image"
+    actual_size: "Actual Size:"
+    image_file_format_header: "Image File Format:"
+    image_file_format_width: "Width: 1 byte"
+    image_file_format_height: "Height: 1 byte"
+    image_file_format_bits_per_pixel: "Bits per Pixel: 1 byte"
+    image_file_format_data_a: "n bits of pixel data"
+    image_file_format_data_b: "n = Width * Height * Bits per Pixel"
+    image_width: "Image Width:"
+    image_height: "Image Height:"
+    bits_per_pixel_header: "Bits Per Pixel:"
+    binary: "Binary:"
+    hexadecimal: "Hexadecimal:"
+    pixel_format_header: "Pixel Format:"
```

Comment:
Kind of a nitpick, but I'm wondering if the `:` should be in the translatable content. It seems kind of like this something structural and should be part of the UI instead. Looking at the screenshots, I feel like the elements would look fine without the `:`.

Reply:
I agree. They aren't even that consistent. I'll just remove them. They aren't adding anything and might interfere with a translation.

Reply:
Removed them!